tensordict/__init__.py

```diff
@@ -5,6 +5,7 @@
 
 from .memmap import MemmapTensor, set_transfer_ownership
 from .tensordict import (
+    _TensorDictKeysView,
```

Comment:
Looks like this got added back in, can we remove from public interface?

Reply:
Done